manifests/v/Velocidex/Velociraptor/0.6.3/Velocidex.Velociraptor.installer.yaml

```
@@ -0,0 +1,21 @@
+# Created with YamlCreate.ps1 v2.0.4 $debug=QUSU.7-2-1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Velocidex.Velociraptor
+PackageVersion: 0.6.3
+InstallerLocale: en-US
+MinimumOSVersion: 10.0.0.0
+InstallerType: wix
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/Velocidex/velociraptor/releases/download/v0.6.3/velociraptor-v0.6.3-windows-386.exe
```

needs to select the msi
```suggestion
  InstallerUrl: https://github.com/Velocidex/velociraptor/releases/download/v0.6.3/velociraptor-v0.6.3-windows-amd64.msi
```